common/p_interaction.cpp

```diff
@@ -2116,20 +2196,19 @@ void P_DamageMobj(AActor *target, AActor *inflictor, AActor *source, int damage,
 		P_KillMobj(source, target, inflictor, false);
 
 		// WDL damage events.
+		// todo: handle voodoo dolls here
```

Comment:
Online voodoo dolls do not take damage.

Reply:
During testing, I observed on Sunlust.WAD MAP02 on a client/server setup the voodoo dolls would throw a "ENVIROKILL" event whenever they were damaged. I thought this was the mechanism that was doing so, but I didn't look into it very long.